### ISSUE TITLE
numexpr appears to be a requirement for 0.2.0

### ISSUE THREAD

**Issue by ctb** [camillescott/dammit#39]
```
TaskError - taskid:fasta-annotate:short.fa.dammit.fasta[dammit.annotate:ERROR]
PythonAction Error
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/doit/action.py", line 383, in execute
    returned_value = self.py_callable(*self.args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/dammit/tasks.py", line 593, in annotate_fasta
    df = annotations.query('seqid == "{0}"'.format(record.name))
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py", line 2107, in query
    res = self.eval(expr, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py", line 2159, in eval
    return _eval(expr, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pandas/computation/eval.py", line 218, in eval
    _check_engine(engine)
  File "/usr/local/lib/python2.7/dist-packages/pandas/computation/eval.py", line 40, in _check_engine
    raise ImportError("'numexpr' not found. Cannot use "
ImportError: 'numexpr' not found. Cannot use engine='numexpr' for query/eval if 'numexpr' is not installed
[dammit.annotate:ERROR] 
```

**Comment by ctb** [comment]
Oddly, pip install dammit didn't install numexpr... despite it being in requirements.txt. I'll come back to this.

**Comment by ctb** [comment]
Oh, I am told this was fixed by 0.2.2.
